src/Client/ClientBase.cpp

```diff
@@ -1709,6 +1708,7 @@ void ClientBase::init(int argc, char ** argv)
         ("profile-events-delay-ms", po::value<UInt64>()->default_value(profile_events.delay_ms), "Delay between printing `ProfileEvents` packets (-1 - print only totals, 0 - print every single packet)")
 
         ("interactive", "Process queries-file or --query query and start interactive mode")
+        ("suggest-interval-ms", po::value<UInt64>()->default_value(suggest_interval_ms), "Reload suggestions interval")
```

Comment:
Also need to add here https://github.com/ClickHouse/ClickHouse/blob/master/src/Client/ClientBase.cpp#L1785?

Reply:
Ok, let's add an ability to specify this delay in client xml config. Done.